spec/controllers/development_metrics_controller_spec.rb

```diff
@@ -44,10 +45,82 @@
       end
 
       context '#projects' do
+        render_views
+
+        subject { get :projects, params: params }
+
+        let(:params) do
+          {
+            project_name: project.name,
+            metric: {
+              metric_name: 'defect_escape_rate',
+              period: 'weekly'
+            }
+          }
+        end
+
         it 'calls CodeClimate summary retriever class' do
           expect(CodeClimateSummaryRetriever).to receive(:call).and_return(code_climate_metric)
 
-          get :projects, params: params
+          subject
+        end
+
+        context 'when it has issues from different environments' do
+          let!(:production_jira_bugs) do
+            create_list(:jira_issue, 2,
+                        :bug,
+                        :production,
+                        jira_project: jira_project,
+                        informed_at: beginning_of_day)
+          end
+          let!(:staging_jira_bugs) do
+            create_list(:jira_issue, 1,
+                        :bug,
+                        :staging,
+                        jira_project: jira_project,
+                        informed_at: beginning_of_day)
+          end
+          let!(:qa_jira_bugs) do
+            create_list(:jira_issue, 3,
+                        :bug,
+                        :qa,
+                        jira_project: jira_project,
+                        informed_at: beginning_of_day)
+          end
+          let!(:no_env_jira_bugs) do
+            create_list(:jira_issue, 2,
+                        :bug,
+                        :no_environment,
+                        jira_project: jira_project,
+                        informed_at: beginning_of_day)
+          end
+
+          before do
+            subject
+          end
+
+          it 'has a successful response' do
+            expect(response).to be_successful
+          end
+
+          it 'render EDR metric with correct production issues' do
+            expect(response.body).to include('Production:' + NEWLINE_WHITESPACES +
```

Comment:
the `NEWLINE_WHITESPACES` thing looks fragile. Why not doing `/Production:\s*#{production_jira_bugs.count}/` ?

Also don't use `+` for string interpolation, instead use `"#{} #{}`